docs/spec_draft.md

```diff
@@ -1779,6 +1780,50 @@ More formally, `results[:][result_index] = reduce(input_slices)` where:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reduce_precision
+
+### Semantics
+
+Reduces floating-point precision element-wise by modeling the effect of
+converting floating-point value to a lower precision format using
+`exponent_bits` and `mantissa_bits` and back to the original format on
+`operand` tensor and produces a `result` tensor. `exponent_bits` and
+`mantissa_bits` can be specified arbitrarily, but not all bit sizes may be
```

Comment:
IMO this line is meant to be provided just for the XLA compiler and  the spec should not care about  the support status of the implementation. One problem would be the spec will become dependent on the evolution of the implementation. 

Note that this is different from implementation defined which says diff implementation can take different stance.